### ISSUE TITLE
Cannot keyboard navigate to new created content

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#9237]
**Case 1.**
- Expand `Large tree`   folder 
- Add one or more child folders
- Press `Shift + Arrow down` keys - try to navigate to the new content. 

**BUG**  - Selecting in content grid using  `Shift+Arrow down`  stop working  after adding new content

<img width="709" height="248" alt="Image" src="https://github.com/user-attachments/assets/94e21f82-92d2-478b-b220-2ff9668b11f0" />

**Case 2** 
-  Create a new folder.    Newly added content is always shown as the top item in the grid. - OK. 
-  Click on the new created folder and press `Arrow Down` key

   **BUG** -    Cannot navigate to the next item after the just created content 

<img width="740" height="538" alt="Image" src="https://github.com/user-attachments/assets/40bbc60a-3ec3-42c9-a337-60e8f0fb7c41" />

Click on the second item  in the grid  and press `Arrow Up` key 

**BUG**  -  Cannot navigate to the new created content  

<img width="712" height="236" alt="Image" src="https://github.com/user-attachments/assets/f125543d-6339-4dcb-ba53-134dbab6e565" />
